source/features/user-local-time.tsx

```diff
@@ -101,7 +101,13 @@ function init(): void {
 
 		// Adding the time element might change the height of the hovercard and thus break its positioning
 		const hovercardHeight = hovercard.offsetHeight;
-		select('div.d-flex.mt-3 > div.overflow-hidden.ml-3:not([data-hovercard-type="contributors"]', hovercard)!.append(container);
+
```

Comment:
```suggestion
		// Github uses the same class for the contributer hover
```

Should I add a comment?